src/main/java/mate/academy/app/service/impl/ReviewServiceImpl.java

```diff
@@ -9,6 +10,7 @@
 @Service
 public class ReviewServiceImpl implements ReviewService {
     private final ReviewRepository reviewRepository;
+    private static final String REGEX = "[^a-z]+";
```

Comment:
REGEX - it's not informative, what kind of regex is that, what does it check/represent?